training_record/services.py

```diff
@@ -79,6 +79,63 @@ def create_off_campus_record_from_raw_data(
                 )
         return record
 
+    # pylint: disable=invalid-name
+    # pylint: disable=redefined-builtin
+    @staticmethod
+    def update_off_campus_record_from_raw_data(
+            record, off_campus_event=None, user=None,
+            contents=None, attachments=None):
+        '''Update the off-campus record'''
```

Comment:
Please add detailed `parameters` and `return` section to describe the signature of this function.